src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_mysql.py

```diff
@@ -425,6 +426,52 @@ def flexible_server_connection_string(
     }
 
 
+def connect_to_flexible_server_mysql(cmd, server_name, administrator_login, administrator_login_password, database_name=None, mysql_query=None):
+    host = '{}.mysql.database.azure.com'.format(server_name)
```

Comment:
the host name for different cloud is the same? 
could we use cloud information in cli ctx to adapt different cloud?

Reply:
This is a good catch, thanks! Fixing in new commit.  I am now using cli_ctx.cloud.suffixes.<mysql/postgres>endpoint.  Confirmed by logging into Mooncake.  Flexible server MySQL/PostgreSQL is not yet deployed in any cloud outside AzureCloud.